### ISSUE TITLE
Is is possible to check special moves?

### ISSUE THREAD

**Issue by 9acca9** [ctabin/jchess#52]
Hi.
Sorry the ignorance.
I made a little app in python using python-chess (a module to chess validation move), with this module you can ask if certain moves are "en passant", "castling", "promotion".
For example:
```
if chess_board.is_castling(move): 
      do...
```
if move is a move that is castling the if will be true

Now im trying to replicate my app but with Java... (im trying to learn java...)
It is possible to know if this moves are happening with your library?

Thanks!

**Comment by corlaez** [comment]
Yeah, it is possible but you would have to do it manually. No convenient methods like you described,  @9acca9

**Comment by 9acca9** [comment]
Thank you very much.
